selendroid-grid-plugin/build.gradle

```diff
@@ -0,0 +1,9 @@
+apply plugin: 'java'
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile 'org.seleniumhq.selenium:selenium-server:2.45.0'
```

Comment:
bump to 2.48.2  ... is there a way to define the selenium server version is a 'root' gradle settings file so we don't have to duplicate it everywhere?